ush/generate_FV3LAM_wflow.py

```diff
@@ -114,21 +113,10 @@ def generate_FV3LAM_wflow(ushdir, logfile: str = "log.generate_FV3LAM_wflow", de
         #
         rocoto_yaml_fp = expt_config["workflow"]["ROCOTO_YAML_FP"]
         args = ["-o", wflow_xml_fp,
-                "-t", template_xml_fp,
-                "-c", rocoto_yaml_fp ]
-        if not debug:
-            args.append("-q")
-
-        try:
-            fill_jinja_template(args)
-        except:
-            raise Exception(
-                dedent(
-                    f"""
-                    Call to fill_jinja_template failed.
-                    """
-                )
-            )
+                "-i", template_xml_fp,
+                "-c", rocoto_yaml_fp,
+                ]
+        set_template(args)
```

Comment:
Does the new replacement script not have an equivalent quiet/verbose flag?

Reply:
I was actually asking about the opposite. The previous script had increased verbosity if the "quiet" flag was not provided, and this was toggled in generate_FV3LAM_wflow.py with the `debug` flag. Is there an equivalent flag to `debug` that we can pass to this new script to get increased verbosity?

Reply:
It looks like this does not exist. Resolving this conversation.

Reply:
I will add an Issue to UW to add some additional debug output. Our `set_config` does have this capability, and it's a good one.